dockerizeit/master/matrixbasedsecurity.groovy

```diff
@@ -0,0 +1,30 @@
+import java.util.Properties
+import java.lang.System
+import jenkins.model.*
+import hudson.security.*
+import org.jenkinsci.plugins.*
+import hudson.model.*
+
+println "--> Matrix-Based security: Read properties from the file"
+
+Properties properties = new Properties()
+def home_dir = System.getenv("JENKINS_HOME")
+File propertiesFile = new File("$home_dir/jenkins.properties")
```

Comment:
Shall we make a function out of that? This is not a call for action. Just asking? If so we can make another ticket to create a function and add it to all files